npc/re/kafras/kafras.txt

```diff
@@ -110,7 +110,10 @@ job3_rune01,92,62,3	script	Kafra Employee::kaf_rune	114,{
 harboro1,291,207,5	script	Kafra Employee::kaf_rock	4_F_KAFRA1,{
 	cutin "kafra_01",2;
 	callfunc "F_KafSet";
-	callfunc "F_Kafra",0,8,2,500,0;
-	savepoint "harboro1",286,189,1,1;
-	callfunc "F_KafEnd",0,1,"in the city of Rockridge";
+	mes "[Kafra Employee]";
+	mes "Welcome to Kafra. We're always by your side.";
+	mes "How may I help you?";
```

Comment:
This looks like the default kafra message:
```
	default:// Default message (obsolete)
		mes "[Kafra Employee]";
		mes "Welcome to the";
		mes "Kafra Corporation.";
		mes "The Kafra services";
		mes "are always on your side.";
		mes "How may I assist you?";
		break;
```

Reply:
After I wrote that I noticed